content/components/text-inputs/valid-invalid.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: component/code-demo
+iframe: examples/example-valid-invalid
+iframeFullwidth: true
+code:
+  - HTML: |
+      <!--
+        Light:  <input class="au-text-input au-text-input--valid">
+        Dark:   <input class="au-text-input au-text-input--valid au-text-input--dark">
+        Invalid:  <input class="au-text-input au-text-input--invalid">
```

Comment:
Include `Invalid dark:` as well.

Reply:
I think just invalid and invalid is good